src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -404,6 +428,18 @@ protected int doWork() {
     private void rejectVariant(final VariantContext ctx, final String reason) {
         rejects.add(new VariantContextBuilder(ctx).filter(reason).make());
         failedLiftover++;
+        trackLiftedVariantContig(rejectsByContig, ctx.getContig());
+    }
+
+    private void trackLiftedVariantContig(Map<String, Long> map, String contig) {
+        Long val = map.get(contig);
+        if (val == null){
+            val = 0L;
+        }
+
+        val++;
+
+        map.put(contig, val);
```

Comment:
Could get rid of lines 439-441 and make this map.put(contig, ++val);

Reply:
ok